src/lib/browse/layers/points/CycleParking.svelte

```diff
@@ -31,6 +29,9 @@
         </ExternalLink> (as of 9 August 2023). The type of parking, public/private
         access, and whether it's covered are not shown.
       </p>
+      <p>
+        Icon from <a href="https://github.com/gravitystorm/openstreetmap-carto?tab=License-1-ov-file#readme">OpenStreetMap Carto</a>
```

Comment:
Probably just https://github.com/gravitystorm/openstreetmap-carto as a link is fine